e2e/common/common.go

```diff
@@ -8,19 +8,31 @@ import (
 	"os/signal"
 	"path"
 	"path/filepath"
+	"strconv"
 	"syscall"
 	"time"
 
+	"github.com/pkg/errors"
+
 	"github.com/loomnetwork/loomchain/e2e/engine"
 	"github.com/loomnetwork/loomchain/e2e/lib"
 	"github.com/loomnetwork/loomchain/e2e/node"
 )
 
+const (
+	loomExeEv       = "LOOMEXE_PATH1"
+	loomExe2Ev      = "LOOMEXE_ALTPATH"
+	ValidatorsEv    = "VALIDATORS"
+	AltValidatorsEv = "ALT_VALIDATORS"
```

Comment:
Both of these should start with a lower case letter since they don't need to be exported from this package.